pom.xml

```diff
@@ -250,15 +250,6 @@
 				</configuration>
 			</plugin>
 
-			<plugin>
```

Comment:
I don't *think* we use this plugin but it could be a dependency for something else we've included in the POM. We'll need to make sure nothing breaks in pipelines etc when this is merged.

Reply:
I'll add the plugin back just in case